src/components/AttachmentModal.js

```diff
@@ -204,8 +217,17 @@ function AttachmentModal(props) {
      */
     const closeConfirmModal = useCallback(() => {
         setIsAttachmentInvalid(false);
+        setIsDeleteCommentConfirmModalVisible(false);
     }, []);
 
+    /**
+     * Close the confirm modal.
```

Comment:
Maybe clarify which modal this closes?

```suggestion
     * Close the confirm detach receipt modal.
```